flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCTableSource.java

```diff
@@ -112,6 +120,11 @@ public boolean isBounded() {
 		return returnType;
 	}
 
+	@Override
+	public DataType getProducedDataType() {
```

Comment:
Please remove the overrided implementation of `getReturnType()`

Reply:
Sure